components/contribution-flow/ContributionFlowSuccess.js

```diff
@@ -304,7 +346,7 @@ class ContributionFlowSuccess extends React.Component {
               </Flex>
             </ContainerWithImage>
             <Flex flexDirection="column" alignItems="center" justifyContent="center" width={1}>
-              {isPendingBankTransfer ? this.renderBankTransferInformation() : this.renderCallsToAction()}
+              {this.renderInformation()}
```

Comment:
Doesn't seem correct to call `renderCallsToAction` from `renderInformation`. You should find a better name.

Reply:
Hmmm, I've renamed it to `addSuccessInfoByPaymentMethod` which to me sounds better. WDYT? 🤔

Reply:
I renamed it to `addSuccessInfoByPaymentMethod` which I think is better, what do you think? 🤔